MANIFEST.in

```diff
@@ -1,2 +1,23 @@
 include README.rst
 include CHANGES
+include LICENSE
+
+# the next few stanzas are for astropy_helpers.  It's derived from the
+# astropy_helpers/MANIFEST.in, but requires additional includes for the actual
+# package directory and egg-info.
+
+include astropy_helpers/README.rst
+include astropy_helpers/CHANGES.rst
+include astropy_helpers/LICENSE.rst
+recursive-include astropy_helpers/licenses *
+
+include astropy_helpers/ez_setup.py
+include astropy_helpers/ah_bootstrap.py
+
+recursive-include astropy_helpers/astropy_helpers *.py *.pyx *.c *.h *.rst
+recursive-include astropy_helpers/astropy_helpers.egg-info *
+# include the sphinx stuff with "*" because there are css/html/rst/etc.
+recursive-include astropy_helpers/astropy_helpers/sphinx *
+
+prune astropy_helpers/build
+prune astropy_helpers/astropy_helpers/tests
```

Comment:
Copied this text verbatim from astropy. Hope that is ok. Happy to add attribution or whatever is necessary.

Reply:
Looks good, though a few more things may be still needed: https://github.com/astropy/package-template/blob/rendered/MANIFEST.in#L10

Reply:
Yes, I think so?  Sorry, I couldn't find this comment earlier.

Reply:
Ok, since this is already merged. Would just copy this `recursive-include docs *` and add it here.

Reply:
Thanks, I'll add that as I'm making the pypi release